bigtable-hbase/src/test/java/com/google/cloud/bigtable/hbase/TestBigtableBufferedMutator.java

```diff
@@ -90,9 +98,12 @@ public void testNoMutation() throws IOException {
   @SuppressWarnings("unchecked")
   @Test
   public void testMutation() throws IOException {
```

Comment:
All we care about here is that the client call to .mutateRowAsync is called rather than the future completing.  The results of that call are out of scope for BigtableBufferedMutator.